src/core/stdc/fenv.d

```diff
@@ -258,6 +258,24 @@ else version ( OpenBSD )
 
     alias fexcept_t = uint;
 }
+else version ( DragonFlyBSD )
+{
+    struct fenv_t
+    {
+        struct _x87
+        {
+                uint control;
+                uint status;
+                uint tag;
+                uint[4] others;
+        };
+        _x87 x87;
+
+        uint mxcsr;
```

Comment:
Not related to this patch, but I've just noticed that the OpenBSD branch is wrong.